src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -70,21 +70,23 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return 1;
         }
 
-        $envFiles = $this->getEnvFiles();
-        $availableFiles = array_filter($envFiles, fn (string $file) => is_file($this->getFilePath($file)));
+        $filePath = $_SERVER['SYMFONY_DOTENV_PATH'];
```

Comment:
This variable is always set if `SYMFONY_DOTENV_VARS` is set. So I don't add a fallback that would be: 

```suggestion
        $filePath = $_SERVER['SYMFONY_DOTENV_PATH'] ?? $this->projectDirectory.'/.env';
```

Reply:
let's add the fallback, we never know how things go

Reply:
Fallback and changelog added.